cg/meta/workflow/mutant/metadata_parser/models.py

```diff
@@ -0,0 +1,15 @@
+from pydantic import BaseModel
+
+
+class SampleMetadata(BaseModel):
+    sample_internal_id: str
+    sample_name: str
+    is_external_negative_control: bool
+    is_internal_negative_control: bool = False
+    reads: int
+    target_reads: int
+    percent_reads_guaranteed: int
+
+
+class SamplesMetadataMetrics(BaseModel):
+    samples: dict[str, SampleMetadata]
```

Comment:
You could consider letting this be 
```suggestion
class SamplesMetadataMetrics(BaseModel):
    samples: list[SampleMetadata]
    internal_sample: SampleMetadata
    external_sample: SampleMetadata
```
or something. Seems as if there is a lot of looping that we might be able to avoid if it was easier to say which samples are controls already in this class.

Reply:
Good idea! Will change everything accordingly.